src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/02-Plugins.md

```diff
@@ -153,10 +142,19 @@ integrate.
 To make an auto plugin, create a project and enable `SbtPlugin`.
 
 ```scala
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "com.example"
+ThisBuild / homepage := Some(url("https://github.com/sbt/sbt-hello"))
```

Comment:
Specifically to address the issue that this PR is fixing, I would comment in the text that here the `scalaVersion` is missing, so that it is not incorrectly set to one that does not match SBT.